bin/terminus

```diff
@@ -21,20 +21,19 @@ if ($phar_path) {
 }
 
 use Pantheon\Terminus\Config;
-use Pantheon\Terminus\Runner;
-use Robo\Robo;
-use Symfony\Component\Console\Application;
+use Pantheon\Terminus\Terminus;
 use Symfony\Component\Console\Input\ArgvInput;
 use Symfony\Component\Console\Output\ConsoleOutput;
 
-// Configuring the dependency-injection container
+// Create the input and output objects for Terminus to run against.
 $input = new ArgvInput($_SERVER['argv']);
 $output = new ConsoleOutput();
+// Create a config object.
 $config = new Config();
-$application = new Application('Terminus', $config->get('version'));
-$container = Robo::createDefaultContainer($input, $output, $application, $config);
 
 // Running Terminus
-$runner = new Runner($container);
-$status_code = $runner->run($input, $output);
+// @TODO: Figure out if and why we need to pass input/output twice and define what happens if we
```

Comment:
@TeslaDethray, @greg-1-anderson Thoughts on this? It seems weird to pass these into the app twice without it being clear why. This is pulled up from the way Robo requires an input/output during setup and run. I'm not sure what would happen if you created new objects between construction and 'run' but if that's not supported or doesn't give meaningful benefits then we should simplify this.

The easiest way would be to simply store the input and output objects on instatiation and then just call `$terminus->run()` but I like the idea of being able to call `run` on terminus with different input (different output I'm less concerned about).

The alternative would be to just pass `$config` to the instantiator and let Robo set up default i/o, then override those on `run`.